data_steward/tools/add_hpo.py

```diff
@@ -200,9 +219,56 @@ def bucket_access_configured(bucket_name):
         raise
 
 
+def update_site_masking_table():
+    """
+    Creates a unique `site_maskings` sandbox table and updates the `pipeline_tables.site_maskings` table with the
+        new site maskings
+    """
+
+    project_id = app_identity.get_application_id()
+    dataset_id = PIPELINE_TABLES
+    intermediary_dataset_id = dataset_id + '_sandbox'
```

Comment:
Does this need to be sandboxed to the dataset it's running on instead of the `pipeline_tables_sandbox`? Not sure if that sandbox exists?

Reply:
It exists! It's created in the `SITE_MAPPINGS_QUERY` in `generate_site_mappings_and_ext_tables.py`. I will change the parameter name from `dataset_id` to `sandbox_dataset_id` in that query to make this more clear.